wdl-lsp/Cargo.toml

```
@@ -27,5 +27,14 @@ tracing = { workspace = true }
 url = { workspace = true }
 uuid = { workspace = true, features = ["v4"] }
 
+[dev-dependencies]
+anyhow = { workspace = true }
+tempfile = { workspace = true }
+serde = { workspace = true }
+serde_json = { workspace = true }
+tokio = { workspace = true, features = ["full"] }
+pretty_assertions = { workspace = true }
+fs_extra = { workspace = true }
```

For the ones that are already in the `dependencies`, you don't need to declare them here again. Further, we should alphabetize this list.

```suggestion
fs_extra = { workspace = true }
pretty_assertions = { workspace = true }
serde = { workspace = true }
tempfile = { workspace = true }
```